package.json

```diff
@@ -55,6 +55,7 @@
     "babel-core": "^6.22.1",
     "babel-loader": "^6.2.10",
     "babel-plugin-istanbul": "^4.1.1",
+    "babel-plugin-remove-comments": "^2.0.0",
```

Comment:
Ahh yes, I see there is a [compact](https://blog.mariusschulz.com/2016/07/12/speeding-up-babel-transpilation-with-compact-mode) option too, worth a try.